src/styles/card.scss

```
@@ -37,49 +48,38 @@
     display: flex;
     justify-content: space-between;
     align-items: center;
-        
+
     margin-top: 15px;
     margin-bottom: 20px;
+  }
 
-    .card__stars {
-      display: flex;
-
-      .star {
-        background: url("../images/star-active.svg") no-repeat center;
-        margin-right: 4px;
-        height: 15px;
-        width: 15px;
-  
-        &:nth-child(5) {
-          background: url("../images/star.svg") no-repeat center;
-        }
-      }
-    }
+  &__stars {
+    display: flex;
+  }
 
-    .card__comments {
-      font-weight: 400;
-      font-size: 11px;
-      line-height: 13px;
-      text-align: right;
-    }
+  &__comments {
+    font-weight: 400;
+    font-size: 11px;
+    line-height: 13px;
+    text-align: right;
   }
 
   &__amount {
     display: flex;
     justify-content: space-between;
     align-items: center;
+  }
 
-    .card__cost {
-      font-weight: 400;
-      font-size: 11px;
-      line-height: 13px;
-    }
+  &__cost {
+    font-weight: 400;
```

Створюй окремі перемінні, для кращої підтримки коду. Наприклад: $fw-normal: 400; $fw-semibold: 500; $fw-bold: 700;
```suggestion
    font-weight: $fw-normal;
```